tools/talis/init.go

```diff
@@ -153,7 +154,7 @@ func CopyTalisScripts(destDir string, srcRoot string) error {
 			return fmt.Errorf("git clone failed: %w", err)
 		}
 
-		src = filepath.Join(tmp, "tools", "talis", "scripts")
+		src = filepath.Join(tmp, "tools", appconsts.TalisChainID, "scripts")
```

Comment:
this one should also be reverted because it's a coincidence, but doesn't need to be the chain ID

```suggestion
		src = filepath.Join(tmp, "tools", "talis", "scripts")
```